src/main/java/racingCar/Car.java

```diff
@@ -0,0 +1,36 @@
+package racingCar;
+
+public class Car {
+    private String carName;
+    private int forwardCnt;
+
+    public Car(String carName){
+        checkCarName(carName);
+        this.carName = carName;
+        this.forwardCnt = 0;
+    }
+
+    public Car(String carName, int forwardCnt){
+        checkCarName(carName);
+        this.carName = carName;
+        this.forwardCnt = forwardCnt;
+    }
+
+    private void checkCarName(String carName){
+        if(carName.length()>5){
```

Comment:
```suggestion
        if(carName.length()> MAX_NAME_LENGTH){
```

매직넘버를 의미를 드러내는 상수화해보아도 좋겠네요!